pages/windows/winget.md

```diff
@@ -15,10 +19,18 @@
 
 `winget search {{package}}`
 
-- Upgrade all packages to latest versions:
+- Upgrade all packages to the latest versions:
 
 `winget upgrade --all`
 
-- List all packages installed that can be managed with winget:
+- List all packages installed that can be managed with `winget`:
 
 `winget list --source winget`
+
+- Import packages from a file, or export installed packages to a file:
+
+`winget {{import|export}} {{--import-file|--output}} {{path/to/file}}`
+
+- Validate manifests before submitting PR to Winget repository:
```

Comment:
I changed `Winget` to `winget-pkgs` because that's the name of the repository.
https://github.com/microsoft/winget-pkgs
```suggestion
- Validate manifests before submitting a PR to the winget-pkgs repository:
```